src/Uno.UWP/Storage/Pickers/FileSavePicker.cs

```diff
@@ -12,6 +12,6 @@ public partial class FileSavePicker
 
 		public string CommitButtonText { get; set; }
 
-		public IDictionary<string, IList<string>> FileTypeChoices { get; set; }
+		public IDictionary<string, IList<string>> FileTypeChoices { get; set; } = new Dictionary<string, IList<string>>();
```

Comment:
In UWP `FileTypeChoices`  is get only: https://docs.microsoft.com/en-us/uwp/api/windows.storage.pickers.filesavepicker.filetypechoices?view=winrt-19041

We should change to `{ get; } = ` or `{ get; private set; } = `

Reply:
Alternatively make the setter obsolete setter (C# 8) as follows:


```c#
private IDictionary<string, IList<string>> fileTypeChoices;
public IDictionary<string, IList<string>> FileTypeChoices
{
  get => fileTypeChoices ?? = new Dictionary<string, IList<string>>(); //or initialize backing field
  [Obsolete("This setter will be removed in future versions.")]
  set = fileTypeChoices =  value;
}
```

Reply:
@weitzhandler This change may not be acceptable without nullable references, as FileTypeChoices may be null and must be validated at all uses sites. If `#nullable enable` is added, then yes, it's useful.

Reply:
@jeromelaban are you talking about the `??=` operator?

Reply:
Yes, if the `fileTypeChoices` member is used elsewhere in the class, it may fail because of a null reference exception.

Reply:
You're right. I was relying that we shall never access the field directly.

Reply:
Alternatively

private readonly IDictionary<string, IList<string>> fileTypeChoices;
public IDictionary<string, IList<string>> FileTypeChoices
{
  get => fileTypeChoices;
  [Obsolete("This setter will be removed in future versions.")]
  set = fileTypeChoices = // copy from value;
}

Reply:
@jeromelaban - I'm assuming I need to merge the PackageDiffIgnore.xml from master into my branch and add the setter in and try this the CI again?